Week1/homework/create-db-tables.js

```diff
@@ -0,0 +1,27 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  multipleStatements: true,
+});
+
+connection.connect();
+
+const sqlCommand = `
+  CREATE DATABASE new_world;
+
+  USE new_world; 
+
+  CREATE TABLE if not exists country(id INT AUTO_INCREMENT PRIMARY KEY, Name VARCHAR(50), Continent VARCHAR(50), Region VARCHAR(50), SurfaceArea FLOAT, IndepYear INT, Population INT, LifeExpectancy FLOAT, GNP FLOAT(7,2), GNPOld FLOAT(7,2), LocalName VARCHAR(50), GovernmentForm VARCHAR(50), HeadOfState VARCHAR(50), Capital VARCHAR(30));
+
+  CREATE TABLE if not exists city(id INT AUTO_INCREMENT PRIMARY KEY, Name VARCHAR(50), CountryCode VARCHAR(10), District VARCHAR(30), Population INT);
+`;
```

Comment:
Good for both id auto increment primary key ! Continent could have been an enum but varchar works also.